maintainer/conda/MDAnalysis/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+MDA_USE_OPENMP=FALSE pip install package/
```

Comment:
Conda doesn't ship openmp libraries. To avoid that a release package is build with globally installed openmp versions we default to only serial builds on conda do ensure that the binary package truly runs anywhere.